modules/sdk/project-templates/project-templates-npm-angular-portlet/src/main/resources/archetype-resources/build.gradle

```diff
@@ -26,11 +24,11 @@ dependencies {
 	compileOnly group: "org.osgi", name: "osgi.cmpn"
 #else
 #if (${liferayVersion.startsWith("7.0")})
-	compileOnly group: "com.liferay.portal", name: "com.liferay.portal.kernel", version: "2.0.0"
+	compileOnly group: "com.liferay", name: "com.liferay.frontend.js.loader.modules.extender.api", version: "1.0.2"
 #elseif (${liferayVersion.startsWith("7.1")})
 	compileOnly group: "com.liferay", name: "com.liferay.frontend.js.loader.modules.extender.api", version: "2.0.2"
-	compileOnly group: "com.liferay.portal", name: "com.liferay.portal.kernel", version: "3.0.0"
 #end
+	compileOnly group: "com.liferay.portal", name: "com.liferay.portal.kernel", version: "2.0.0"
```

Comment:
Kernel 2.0.0 should be for 7.0 and kernel 3.0.0 should be for 7.1 right?

Reply:
Ivan changed this in the blade samples so that both can compile off kernel 2.0.0. I think keeping it low helps us keep these templates simpler.